### PR TITLE
Desktop-view bug fixes

### PR DESCRIPTION
**Pull request by kiraigg**

### CODE REVIEW

File: client/src/components/styles/_forms.scss [hunk 1]
```diff
@@ -11,13 +11,19 @@
   .forgot-password-form {
     // sass-lint:disable-block nesting-depth
     .btn-primary {
+
+      &:active:focus {
```
Comment (anna-kuzii) [inline client/src/components/styles/_forms.scss]: Strange code. Should be comma here?

File: client/src/components/styles/_forms.scss [hunk 2]
```diff
@@ -81,6 +87,15 @@
   height: 100%;
   max-width: 800px;
 
+  input {
+    padding: 0 15px 0 15px;
```
Comment (anna-kuzii) [inline client/src/components/styles/_forms.scss]: Make it simpler

File: client/src/components/styles/_forms.scss (hunk 1 above)
Comment (kiraigg) [reply client/src/components/styles/_forms.scss]: nope, it's showed only when two states are active at the same time

Comment (anna-kuzii) [reply client/src/components/styles/_forms.scss]: Than use scss format

Comment (kiraigg) [reply client/src/components/styles/_forms.scss]: done

**Pull request merged**